pkg/minikube/download/docker_env.go

```diff
@@ -0,0 +1,42 @@
+package download
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+// DownloadNerdctl downloads nerdctl to directory specified by the param
+func DownloadNerdctl(dir string) error {
+	//check whether nerdctl exists
+	if _, err := os.Stat(filepath.Join(dir, "nerdctl")); err == nil {
+		//file exists
+		return nil
+	}
+	fileName := filepath.Join(dir, "nerdctl-1.0.0-linux-amd64.tar.gz")
+	if err := download("https://github.com/containerd/nerdctl/releases/download/v1.0.0/nerdctl-1.0.0-linux-amd64.tar.gz", fileName); err != nil {
```

Comment:
Even if this is only supported by Linux this will not work on arm64. Should we also support Mac/Windows?

Reply:
probably not. For windows, docker daemon is running in WSL2 instead of the bare machine. (I am not sure how docker is runned on mac), so there may be some problems communicating with the daemon; Besides, we use ssh-agent to login the virtual machine without password, and this is not unavailable on windows, so maybe we should not support mac/windows currently?

Reply:
Starting with Linux is fine with me, but the code is not currently guarding against Mac/Windows, so it's going to attempt to download and start the Linux nerdctl binary on Mac/Windows so we need to prevent this. I think we should also support arm64 on Linux as all that has to change is download the arm64 binary instead.

Reply:
This will only ever be installed on the node (there is no client), so there are no binaries for other OS than Linux